server/src/api/v1/admin/service-hours/index.ts

```diff
@@ -26,7 +26,7 @@ router.post('/', async (req, res) => {
 
 router.delete('/:slotId', async (req, res) => {
   const { slotId } = req.params;
-  let slots = [];
+  let slots: number = null;
```

Comment:
should be array (as the previous default value)

Reply:
is ok, slots is number of deleted slots.